Assets/Scripts/PlayerController/PlayerHandler.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerHandler : MonoBehaviour, IInteractor
+{
+    [SerializeField]
+    private PlayerMovement m_PlayerMovement;
+
+    [SerializeField]
+    private PlayerAnimation m_PlayerAnimation;
+
+    [SerializeField]
+    private PowerupActor m_PowerupActor;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        m_PlayerAnimation = GetComponent<PlayerAnimation>();
+        m_PlayerMovement = GetComponent<PlayerMovement>();
+        m_PowerupActor = GetComponent<PowerupActor>();
+    }
+
```

Comment:
This class should actually just be the PlayerNetworkHandler renamed. I can rename it to PlayerActor instead

Reply:
Ok!